Docs/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,32 +1,36 @@
 ## Fixes issue
+
 Closes: #issue_number
 Addresses: #issue_number (if multiple persons are assigned, the issue)
+Use one of fixes or addresses and remove the other one.
```

Comment:
Add this as a comment, like this
 \<!---  Use one of fixes or addresses and remove the other one.  -->
<!---  Use one of fixes or addresses and remove the other one.  -->